components/TestLibraryStepper/TestLibraryStepper.tsx

```diff
@@ -0,0 +1,147 @@
+﻿import React, {useState} from "react";
+import {Step, StepConnector, StepIconProps, StepLabel, Stepper, Typography} from '@material-ui/core';
+import {createMuiTheme, makeStyles, MuiThemeProvider} from '@material-ui/core/styles';
+import Router from "next/router";
+import {withStyles} from "@material-ui/styles";
+import clsx from "clsx";
+import {Check, MoreHoriz} from "@material-ui/icons";
+import {CandidateTestStatus} from "../../pages/api/candidateApiClient.module";
+
+interface TestLibraryStepperProps {
+    candidateTestStatus: CandidateTestStatus[];
+}
+
+function getSteps() {
+    //TODO: mock data, set number of steps based on test range, maybe props.candidateTestStatus.tests.length?
+    const testNumber = 3;
+
+    //set labels for steps
+    const testLabelArray = [];
+    while (testLabelArray.length + 1 <= testNumber) {
+        testLabelArray.push(`Test ${testLabelArray.length + 1}`);
+    }
+    return testLabelArray;
+}
+
+export default function TestLibraryStepper(props: TestLibraryStepperProps): JSX.Element {
+    const steps = getSteps();
+    //TODO this is example data, set active step with candidate number of results, maybe props.candidateTestStatus.results.length?
+    const activeStep = 2;
+
+    //overrides for material ui stepper styles
+    const testSwitchTheme = createMuiTheme({
+        palette: {
+            primary: {
+                main: 'rgb(255, 193, 0)'
+            }
+        },
+    });
+
+    //overrides for material ui icon styles
+    const TestSwitchIconStyles = makeStyles({
+        root: {
+            backgroundColor: '#eaeaf0',
+            zIndex: 1,
+            color: '#fff',
+            fontSize: 18,
+            width: 30,
+            height: 30,
+            display: 'flex',
+            borderRadius: '50%',
+            justifyContent: 'center',
+        },
+        active: {
+            color: 'rgb(255, 193, 0)',
+            fontSize: 18,
+        },
+        current: {
+            color: 'rgb(255, 255, 255)',
+            zIndex: 0,
+            fontSize: 18,
+            width: 30,
+            height: 30,
+            borderRadius: '50%',
+            backgroundColor: 'rgb(255, 193, 0)',
+        },
+        completed: {
+            color: 'rgb(255, 255, 255)',
+            zIndex: 0,
+            fontSize: 18,
+            width: 30,
+            height: 30,
+            borderRadius: '50%',
+            backgroundColor: 'rgb(255, 193, 0)',
+        },
+    });
+
+    function TestSwitchStepIcon(props: StepIconProps) {
+        const classes = TestSwitchIconStyles();
+        const {active, completed} = props;
+        return (
+            <div
+                className={clsx(classes.root, {
+                    [classes.active]: active,
+                })}
+            >
+                {completed ? <Check className={classes.completed}/> : <MoreHoriz className={classes.current}/>}
+            </div>
+        );
+    }
```

Comment:
Could we split this and it's styles out into a separate component?
Might just make it a little easier to see what's happening! :)

Reply:
I've just split the overrides into their own file an imported them. It seems to be working ok now.